user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -260,7 +243,7 @@ def _set_savings_calculations_flag(self, enable_flag: bool):
                 # set it to none
                 self.logger.info('Filtering criteria `filter_apps` will be reset to NONE because savings '
                                  'estimates are disabled')
-                self.ctxt.set_ctxt('filterApps', QualFilterApp.NONE)
+                self.ctxt.set_ctxt('filterApps', QualFilterApp.ALL)
```

Comment:
Curios why it was changed to ALL. Should we update the above log comment as well?

Reply:
Good catch regarding updating the log message. I followed up changing all the docs/logs accordingly

`QualFilterApp.NONE` was a bad choice at the beginning;

1. especially as it is part of user's input. This becomes ambiguous with the python-None/none. example, If a user inputs `--filter_aps=None`
2. I saw that the core filter's argument refers to selecting all apps as `all`. `ALL` makes more sense in this context because it means `show me all apps` Vs `show me apps with speedups`..etc.